README.md

```diff
@@ -127,13 +35,16 @@ composer require digiaonline/graphql:dev-master
 
 ### Example
 
-This script demonstrates the public API:
+Here is a simple example that demonstrates how to build an executable schema from a GraphQL schema file that contains 
+the Schema Definition Language (SDL) for a Star Wars -themed schema (for the schema definition, see below). Once
```

Comment:
No space before the hyphen in English:

"Star Wars -themed"
->
"Star Wars-themed"

Reply:
Ok, I will fix this.